packages/sanity/src/core/releases/tool/overview/ReleasesOverview.tsx

```diff
@@ -177,6 +177,7 @@ export function ReleasesOverview() {
             <Table<TableBundle>
               // for resetting filter and sort on table when mode changed
               key={bundleGroupMode}
+              defaultSort={{column: '_createdAt', direction: 'desc'}}
```

Comment:
This should preferably be a constant and moved out of the component so we avoid creating new objects on every render